packages/agreement-process/src/services/agreementService.ts

```diff
@@ -704,3 +724,33 @@ export async function addConsumerDocumentLogic(
     agreement.metadata.version
   );
 }
+
+export async function removeAgreementConsumerDocument(
```

Comment:
For consistency, maybe rename in `removeAgreementConsumerDocumentLogic`?

Also, to declutter this file, we could move the two `addConsumerDocumentLogic` and `removeConsumerDocumentLogic` in an `agreementConsumerDocumentProcessor.ts` file